config/eol.yaml

```diff
@@ -24,6 +24,8 @@ elasticsearch:
     eol: 2018-07-31
   - version: '6.5'
     eol: 2020-05-14
+  - version: '7.2'
```

Comment:
Should be 7.5. Not sure if EOL date changes too with this

Reply:
Good catch, fixed including date.